cmd/describe/cluster/cmd.go

```diff
@@ -291,7 +291,14 @@ func run(cmd *cobra.Command, argv []string) {
 	}
 	// Print short cluster description:
 	fmt.Print(str)
-	fmt.Println()
+	limitedSupportReasons, err := r.OCMClient.GetLimitedSupportReasons(cluster.ID())
```

Comment:
NP: We can concatenate the limited support reasons to the `str` variable before line `293`.